test/library/packages/TOML/test/time.chpl

```diff
@@ -0,0 +1,16 @@
+
+use TOML;
+
+config const str: string = """[owner]
```

Comment:
Could you add a test for a time with decimals? e.g. `timestamp2 = 00:32:00.999999`

Reply:
Actually I haven't added fractional second support. I'll add that too.. :+1:

Reply:
Heya, I've made the changes. @ben-albrecht please review it.